### PR TITLE
Boba test

### PR DESCRIPTION
**Pull request by Oscar-Pepper**

### CODE REVIEW

**Review by fluidvanadium — CHANGES_REQUESTED** [review]
the logs assert that the tests pass even when this test fails.

```
Time: 27.401
There was 1 failure:
1) BobaTest(org.ZingoLabs.Zingo.BobaTest)
expected:
    test
but was:
    {
      "sapling_balance": 0,
      "verified_sapling_balance": 0,
      "spendable_sapling_balance": 0,
      "unverified_sapling_balance": 0,
      "orchard_balance": 0,
      "verified_orchard_balance": 0,
      "spendable_orchard_balance": 0,
      "unverified_orchard_balance": 0,
      "transparent_balance": 0
    }
        at org.ZingoLabs.Zingo.BobaTest.BobaTest(BobaTest.kt:38)

FAILURES!!!
Tests run: 2,  Failures: 1


INSTRUMENTATION_CODE: -1

Test reports saved: android/app/build/outputs/integration_test_reports/x86_64

Integration tests PASSED

Waiting for emulator-5554 to shutdown...
Successfully shutdown emulator-5554.
```

**Comment by Oscar-Pepper** [comment]
> the logs assert that the tests pass even when this test fails.
> 
> ```
> Time: 27.401
> There was 1 failure:
> 1) BobaTest(org.ZingoLabs.Zingo.BobaTest)
> expected:
>     test
> but was:
>     {
>       "sapling_balance": 0,
>       "verified_sapling_balance": 0,
>       "spendable_sapling_balance": 0,
>       "unverified_sapling_balance": 0,
>       "orchard_balance": 0,
>       "verified_orchard_balance": 0,
>       "spendable_orchard_balance": 0,
>       "unverified_orchard_balance": 0,
>       "transparent_balance": 0
>     }
>         at org.ZingoLabs.Zingo.BobaTest.BobaTest(BobaTest.kt:38)
> 
> FAILURES!!!
> Tests run: 2,  Failures: 1
> 
> 
> INSTRUMENTATION_CODE: -1
> 
> Test reports saved: android/app/build/outputs/integration_test_reports/x86_64
> 
> Integration tests PASSED
> 
> Waiting for emulator-5554 to shutdown...
> Successfully shutdown emulator-5554.
> ```

good spot, i didnt understand the fail conditions properly... currently we only get PASSED when we dont get a crash.. i will open a new PR to make that change now!

**Review by fluidvanadium — DISMISSED** [review]

**Comment by fluidvanadium** [comment]
testing again

File: android/app/src/androidTest/java/org/ZingoLabs/Zingo/BobaTest.kt [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    fun BobaTest() {  
+        val server = "https://testnet.lightwalletd.com:9067"
+        val seed = "very airport kite ordinary pelican there like spider kitchen top meat zone subway web recycle original merge key depth popular broom snack gaze stumble"
+        val birthday = "2311722"
+        val datadir = MainApplication.getAppContext()!!.filesDir.path
+
+        RustFFI.initfromseed(server, seed, birthday, datadir)
+
+        var balance = RustFFI.execute("balance", "")
+        System.out.println("Balance:")
```
Comment (fluidvanadium) [inline android/app/src/androidTest/java/org/ZingoLabs/Zingo/BobaTest.kt:33]: unfortunately i cant find these log messages from the test in `android/app/build/outputs/integration_test_reports

Comment (Oscar-Pepper) [reply android/app/src/androidTest/java/org/ZingoLabs/Zingo/BobaTest.kt:33]: Try grepping the logcat.txt for 'system.out' I put the exact command I use in chat couple days ago

Comment (Oscar-Pepper) [reply android/app/src/androidTest/java/org/ZingoLabs/Zingo/BobaTest.kt:33]: i ussually ```cd``` into the ```android/app/build/outputs/integration_test_reports``` directory
then i set this up in a terminal while im running tests:
```tail -F x86_64/logcat.txt | grep 'System.out'```

**Comment by Oscar-Pepper** [comment]
closed for #462

**Pull request closed without merging**